flow360/component/simulation/meshing_param/volume_params.py

```diff
@@ -99,6 +99,22 @@ def _validate_single_instance_in_entity_list(cls, values):
             )
         return values
 
+    @pd.field_validator("entities", mode="after")
+    @classmethod
+    def _validate_cylinder_name_length(cls, values):
+        """
+        Check the name length for the cylinder entities due to the 32-character
+        limitation of all data structure names and labels in CGNS format.
+        The current prefix is 'rotatingBlock-' with 14 characters.
+        """
+        # pylint: disable=protected-access
+        for entity in values.stored_entities:
+            if len(entity.name) > 18:
```

Comment:
Can you compute the value 18 instead of hardcode it here? It will be easier to understand for future developers and also easier to change should we decide to change prefix.

Reply:
Sure, added